<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>400</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Firewall</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>254</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>SIP/MASK </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sourceIPInput"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>DIP/MASK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="destIPInput"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>S PORT     </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sourcePortInput"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>D PORT     </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="destPortInput"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>PROTOCOL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="protocolComboBox">
           <item>
            <property name="text">
             <string>ANY</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>TCP</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>UDP</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ICMP</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="permit">
           <property name="text">
            <string>PERMIT</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="deny">
           <property name="text">
            <string>DENY</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="writeLogChecked">
         <property name="text">
          <string>LOG</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_firewall_rule_Btn">
         <property name="text">
          <string>ADD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_firewall_rule_Btn">
         <property name="text">
          <string>DELETE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear_firewall_rule_Btn">
         <property name="text">
          <string>CLEAR</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tableWidget_firewall">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>730</width>
        <height>300</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>266</x>
        <y>310</y>
        <width>721</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Default action</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="D_PERMIT">
         <property name="text">
          <string>PERMIT</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="D_DENY">
         <property name="text">
          <string>DENY</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="change_default_action_Btn">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rewrite_default_firewall_rule_Btn">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>NatList</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_nat">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>880</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="nat_start_Btn">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>10</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Nat start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nat_stop_Btn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>50</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Nat stop</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>LinkList</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_link">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>1010</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
